internal/flags/apiflags/flags_test.go

```diff
@@ -0,0 +1,295 @@
+package apiflags
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/spf13/pflag"
+	"gotest.tools/assert"
```

Comment:
Change this to use the testify assert `github.com/stretchr/testify/assert"` we already pull this in `go.mod`.